app/code/Magento/SalesGraphQl/README.md

```diff
@@ -0,0 +1,6 @@
+# SalesGraphQl
+
+**SalesGraphQl** provides type and resolver information for the GraphQl module
+to generate sales orders information endpoints.
```

Comment:
endpoints => queries
endpoints is REST terminology,
queries and mutations is used in GraphQL